run_tests.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+TEMP=$(date +%s)
+
+if [ -z $TAG ]
+then
+TAG="latest"
+fi
+
+if [ -z $TEST_IMAGE_NAME ]
+then
+TEST_IMAGE_NAME="supervisor-tests:${TEMP}"
+fi
+
+set -ex
+
+docker build -t $TEST_IMAGE_NAME -f Dockerfile.tests --build-arg CACHEBUST=$TEMP .
+
```

Comment:
Please get an ack from @mitchharpur or @miteshvp

Reply:
Our stress tests, n/w tests, etc... can probably be executed here

Reply:
Hey Anmol, I did explained the reason to @miteshvp .

The workflow is now like : 
1.) Build the image from Dockerfile.tests .
2.) If the build happens successfully, the test suite passed.
3.) The npm install in the Dockerfile.tests is inclusive of the testing and development utilitiesm which we don't need in the production environment.
4.) Now, since the image is not useful for us, as of now, we purge it to clear the memory space.

5.) We build the production ready fresh image in the Dockerfile and use npm install --production. Only the production required stuff goes in the container.

Apologies for the long comment. Let me know what do you think.

Regarding the stress testing, we can raise a seperate issue and PR when the time comes.

Reply:
Ok what I am trying to say is, if your intention is to have npm test run via docker image creation, we can as well do it without image creation right?
So this is only meaningful if we are doing some tests against the image or the image instantiated(running container)..
However, we'll take it in its current form and track the tests addition part as part of https://github.com/fabric8-hdd/supervisor/issues/42 ..
Please take care of my other comments and lets close this PR ASAP